Assets/Scripts/Crafting/CraftingManager.cs

```diff
@@ -0,0 +1,150 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class CraftingManager : MonoBehaviour
+{
+    public static CraftingManager Instance;
+
+    [SerializeField] Ingredient currentIngredient;
```

Comment:
I'm not sure we want to provide access to this variable. If this is for debugging, you can use [ReadOnly] instead of [SerializedField]/

Reply:
it is, will do